flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/adaptive/AdaptiveSchedulerTest.java

```diff
@@ -1992,4 +2035,120 @@ public DummyState getState() {
             }
         }
     }
+
+    private static class ExceptionHistoryTester {
```

Comment:
Would it be possible to use this for as a replacement for `org.apache.flink.runtime.scheduler.adaptive.AdaptiveSchedulerTest#runExceptionHistoryTests(java.util.function.BiConsumer<org.apache.flink.runtime.scheduler.adaptive.AdaptiveScheduler,java.util.List<org.apache.flink.runtime.executiongraph.ExecutionAttemptID>>, java.util.function.Consumer<org.apache.flink.runtime.scheduler.adaptive.AdaptiveSchedulerBuilder>, java.util.function.Consumer<org.apache.flink.runtime.jobgraph.JobGraph>)` as well?

Reply:
Totally, I just missed that! Thanks!